SQL/0000-00-00-schema.sql

```diff
@@ -214,8 +214,7 @@ CREATE TABLE `users` (
   `Pending_approval` enum('Y','N') default 'Y',
   `Doc_Repo_Notifications` enum('Y','N') default 'N',
   PRIMARY KEY  (`ID`),
-  UNIQUE KEY `Email` (`Email`),
-  UNIQUE KEY `UserID` (`UserID`)
+  UNIQUE KEY `Email` (`Email`)
```

Comment:
@intralizee the changes to this file all over the place I would ask to rebase this PR without any changes to the SQL schema (except necessary ones) and submit  the others in a clean PR

Reply:
Yah a mixup happened. I was on another PR than this one. After I pulled some unnecessary additions from minor when trying to recover the changes. Also with the single quotes you mentioned. :)